config.sh

```diff
@@ -36,31 +38,53 @@ install-tools () {
 	brew tap cloudfoundry/tap
 	brew install cf-cli
 	cf install-plugin https://static-ice.ng.bluemix.net/ibm-containers-mac
+    bx plugin install container-service -r Bluemix
+}
+
+config-cli () {
+    printf "Installing Kubernetes CLI...\n"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    printf "\nInstalling IBM Cloud Container Service plugin\n"
+    bx plugin install container-registry -r Bluemix
+    bx plugin list
 }
 
 login () {
 	echo "Setting api and login tools."
-	cf api https://api.ng.bluemix.net
-	cf login
-	cf ic login
+	bx login --sso -a api.eu-gb.bluemix.net
```

Comment:
--sso applies only to internal users who are using federated ID - fine for our testing purposes, but must be removed.

Is there a reason why `eu-gb` is now default?

Reply:
I would like the api defintion here, and the registry definitions to be dynamic - change it at the top of the file.